src/renderer/features/network/NetworksList/model/active-networks-model.ts

```diff
@@ -0,0 +1,31 @@
+import { createEvent, createStore, combine, sample } from 'effector';
+
+import { networkUtils, networkModel } from '@entities/network';
+import { Chain } from '@shared/core';
+import { networksListUtils } from '../lib/networks-list-utils';
+
+const networksChanged = createEvent<Chain[]>();
+
+const $networks = createStore<Chain[]>([]);
+
+const $activeNetworks = combine(
+  {
+    networks: $networks,
+    connectionStatuses: networkModel.$connectionStatuses,
+    connections: networkModel.$connections,
+  },
+  ({ networks, connectionStatuses, connections }) => {
+    return networksListUtils
+      .getExtendedChain(networks, connections, connectionStatuses)
+      .filter((o) => networkUtils.isEnabledConnection(o.connection));
+  },
+);
+
+sample({ clock: networksChanged, target: $networks });
+
+export const activeNetworksModel = {
+  $activeNetworks,
+  events: {
```

Comment:
In some places we have empty line before `events`, I think we should have one way for it

Reply:
Thanks, this one I can probably remember in the future, it has actually triggered me to see this empty line :sweat_smile:. It'd be great to have eslint rules for this.. but I know it's a rabbit hole.